<?xml version="1.0" ?>
<sdf version="1.7">
  <world name="default">
    <gravity>0 0 -9.8</gravity>
    <magnetic_field>6e-06 2.3e-05 -4.2e-05</magnetic_field>
    <atmosphere type="adiabatic"/>
    <max_contacts>100</max_contacts>
    <physics type="ode">
      <max_step_size>0.001</max_step_size>
      <real_time_factor>1</real_time_factor>
      <real_time_update_rate>1000</real_time_update_rate>
      <constraints>
        <cfm>0.2</cfm>
        <erp>0.2</erp>
        <contact_max_correcting_vel>10s</contact_max_correcting_vel>
        <contact_surface_layer>0.0001</contact_surface_layer>
      </constraints>
    </physics>
    <scene>
      <ambient>0.4 0.4 0.4 1</ambient>
      <background>0.7 0.7 0.7 1</background>
      <shadows>1</shadows>
    </scene>
    <spherical_coordinates>
      <surface_model>EARTH_WGS84</surface_model>
      <latitude_deg>0</latitude_deg>
      <longitude_deg>0</longitude_deg>
      <elevation>0</elevation>
      <heading_deg>0</heading_deg>
    </spherical_coordinates>
    <include>
      <uri>model://ground_plane</uri>
    </include>
    <include>
      <uri>model://sun</uri>
    </include>
    <include>
      <uri>model://Table</uri>
      <name>table</name>
      <pose>0 0 0 0 0 0</pose>
      <static>1</static>
    </include>
    <include>
      <uri>model://cabinet_1076</uri>
      <name>cabinet</name>
      <pose>-1.45 -0.2 0.693 0 0 -3.1416</pose>
      <scale>1 1 1</scale>
      <static>0</static>
    </include>
    <model name="basket_with_fabric_liner">
      <link name="link">
        <inertial>
          <pose>0.000919 -0.000777 0.076644 -1.61754 1.40943 1.52439</pose>
          <mass>2.51241</mass>
          <inertia>
            <ixx>0.0155308</ixx>
            <ixy>5.96444e-20</ixy>
            <ixz>3.28155e-20</ixz>
            <iyy>0.0149572</iyy>
            <iyz>-1.56873e-20</iyz>
            <izz>0.00922213</izz>
          </inertia>
        </inertial>
        <collision name="collision">
          <geometry>
            <mesh>
              <uri>model://Threshold_Basket_Natural_Finish_Fabric_Liner_Small/meshes/model.obj</uri>
              <scale>1 1 1</scale>
            </mesh>
          </geometry>
          <surface>
            <friction>
              <ode>
                <mu>10</mu>
                <mu2>10</mu2>
              </ode>
              <torsional>
                <coefficient>10</coefficient>
                <use_patch_radius>1</use_patch_radius>
                <patch_radius>0.1</patch_radius>
                <ode/>
              </torsional>
            </friction>
            <contact>
              <ode>
                <max_vel>0.001</max_vel>
                <min_depth>0.001</min_depth>
              </ode>
            </contact>
            <bounce/>
          </surface>
          <max_contacts>10</max_contacts>
        </collision>
        <visual name="visual">
          <geometry>
            <mesh>
              <uri>model://Threshold_Basket_Natural_Finish_Fabric_Liner_Small/meshes/model.obj</uri>
              <scale>1 1 1</scale>
            </mesh>
          </geometry>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <pose>-0.223049 0.297071 0.813228 -0.003546 0.011741 -2.9e-05</pose>
    </model>
    <model name="orange">
      <static>0</static>
      <link name="link">
        <inertial>
          <pose>-0.007138 -0.018373 0.033571 -2.03531 0.337914 2.9756</pose>
          <mass>0.104983</mass>
          <inertia>
            <ixx>5.81552e-05</ixx>
            <ixy>-4.73488e-21</ixy>
            <ixz>-3.16407e-21</ixz>
            <iyy>5.6808e-05</iyy>
            <iyz>-1.00795e-20</iyz>
            <izz>5.65271e-05</izz>
          </inertia>
        </inertial>
        <collision name="collision">
          <geometry>
            <mesh>
              <scale>1 1 1</scale>
              <uri>model://017_orange/google_16k/textured_coacd_0.02.obj</uri>
            </mesh>
          </geometry>
          <surface>
            <friction>
              <ode>
                <mu>10</mu>
                <mu2>10</mu2>
              </ode>
              <torsional>
                <coefficient>10</coefficient>
                <use_patch_radius>1</use_patch_radius>
                <patch_radius>0.1</patch_radius>
                <ode/>
              </torsional>
            </friction>
            <contact>
              <ode>
                <max_vel>0.001</max_vel>
                <min_depth>0.001</min_depth>
              </ode>
            </contact>
            <bounce/>
          </surface>
          <max_contacts>10</max_contacts>
        </collision>
        <visual name="visual">
          <geometry>
            <mesh>
              <scale>1 1 1</scale>
              <uri>model://017_orange/google_16k/textured.obj</uri>
            </mesh>
          </geometry>
          <material>
            <script>
              <uri>model://017_orange/orange.material</uri>
              <name>orange</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <pose>-0.069084 -0.312195 0.815503 0.003887 0.011289 1.4e-05</pose>
    </model>
    <model name="toy_bus">
      <link name="link">
        <inertial>
          <pose>-0.000212 0.001364 0.018716 1.49232 0.031967 -3.13917</pose>
          <mass>0.1</mass>
          <inertia>
            <ixx>7.92631e-05</ixx>
            <ixy>2.89479e-21</ixy>
            <ixz>-6.87841e-21</ixz>
            <iyy>7.65851e-05</iyy>
            <iyz>-8.66667e-22</iyz>
            <izz>1.80489e-05</izz>
          </inertia>
        </inertial>
        <collision name="collision">
          <geometry>
            <mesh>
              <scale>1 1 1</scale>
              <uri>model://SCHOOL_BUS/meshes/model.obj</uri>
            </mesh>
          </geometry>
          <surface>
            <friction>
              <ode>
                <mu>10</mu>
                <mu2>10</mu2>
              </ode>
              <torsional>
                <coefficient>10</coefficient>
                <use_patch_radius>1</use_patch_radius>
                <patch_radius>0.1</patch_radius>
                <ode/>
              </torsional>
            </friction>
            <contact>
              <ode>
                <max_vel>0.001</max_vel>
                <min_depth>0.001</min_depth>
              </ode>
            </contact>
            <bounce/>
          </surface>
          <max_contacts>10</max_contacts>
        </collision>
        <visual name="visual">
          <geometry>
            <mesh>
              <scale>1 1 1</scale>
              <uri>model://SCHOOL_BUS/meshes/model.obj</uri>
            </mesh>
          </geometry>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <pose>0.082475 -0.061716 0.83037 0.199767 -1.55924 -1.03876</pose>
    </model>
  </world>
</sdf>